app/views/users/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="container">
+  <h2><%= @user.email %></h2>
+    <% @wikis.each do |wiki| %>
+    <ul>
+      <li><%= link_to wiki.title, @wiki %></li>
+    </ul>
+    <% end %>
+</div>
+<br />
+<div class="container">
+  <%= render partial: "charges/form" %>
```

Comment:
@bjblock - It's been awhile since I used partials, do I need to pass locals here in the partial since we're using a charges partial in a user view page?